.github/workflows/ci.yaml

```diff
@@ -49,3 +55,16 @@ jobs:
 
       - name: Run integration tests
         run: tox -vve integration
+
+      - name: Send email on failure
```

Comment:
This would mean that an email will be sent when the job fails on a PR, which wouldn't be uncommon. We should only run this step for the periodic job.

Reply:
Run condition updated.